travis-setup.sh

```diff
@@ -14,10 +14,12 @@ sudo bash Miniconda3-latest-${tag}-x86_64.sh -b -p /anaconda
 sudo chown -R $USER /anaconda
 export PATH=/anaconda/bin:$PATH
 
-# Add channels in the specified order.
-for channel in $(grep -v "^#" bioconda_utils/channel_order.txt); do
-    conda config --add channels $channel
-done
+# TODO: it would be nice to have a single location where channels are
+# configured across bioconda-recipes and bioconda-utils.
+conda config --add channels conda-forge
```

Comment:
We could put this into a isolated shell script and call/download it from remote.